storage.rb

```diff
@@ -0,0 +1,93 @@
+require 'json'
+require './app'
+require './book'
+require './teacher'
+require './student'
+require './rentals'
+require './person'
+
+class Storage
+  def initialize(app)
+    @app = app
+  end
+
+  def save_people
+    return if @app.people.empty?
+
+    people_json = @app.people.map(&:as_json)
+    File.write('people.json', JSON.dump(people_json))
+  end
+
+  def load_people
+    # handle case when people.json is not available
+    return unless File.exist?('people.json')
+
+    people_json = JSON.parse(File.read('people.json'))
```

Comment:
- [ ] 👋 Hey there! Just a friendly heads-up! It seems like we might have missed a step in our app. When users exit the app, it'd be super cool if we could save their data into those nifty JSON files 📦, and when they hop back in, we'd want to make sure we load that data up from those files 🚀. Right now, it looks like our app isn't doing that. Let's see if we can fix it up! 😊🛠️